app/org/sagebionetworks/bridge/services/backfill/UserConsentBackfill.java

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.bridge.services.backfill;
+
+import java.util.Iterator;
+
+import org.sagebionetworks.bridge.dao.AccountDao;
+import org.sagebionetworks.bridge.dao.UserConsentDao;
+import org.sagebionetworks.bridge.models.accounts.Account;
+import org.sagebionetworks.bridge.models.accounts.HealthId;
+import org.sagebionetworks.bridge.models.backfill.BackfillTask;
+import org.sagebionetworks.bridge.services.HealthCodeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component("userConsentEventBackfill")
+public class UserConsentBackfill extends AsyncBackfillTemplate {
+
+    private AccountDao accountDao;
+    private UserConsentDao userConsentDao;
+    private HealthCodeService healthCodeService;
+    
+    @Autowired
+    private final void setAccountDao(AccountDao accountDao) {
+        this.accountDao = accountDao;
+    }
+    @Autowired
+    private final void setUserConsentDao(UserConsentDao userConsentDao) {
+        this.userConsentDao = userConsentDao;
+    }
+    @Autowired
+    public void setHealthCodeService(HealthCodeService healthCodeService) {
+        this.healthCodeService = healthCodeService;
+    }
+    
+    @Override
+    int getLockExpireInSeconds() {
+        return 30 * 60;
+    }
+
+    @Override
+    void doBackfill(BackfillTask task, BackfillCallback callback) {
+        callback.newRecords(getBackfillRecordFactory().createOnly(task, "Starting to examine accounts"));
+        
+        Iterator<Account> i = accountDao.getAllAccounts();
```

Comment:
This is a one time task. I iterate over all accounts every week for that status report, it doesn't fail, but if it did, it wouldn't be the end of the world to start over again. It takes about an hour. It's not worth it to me to invest to much effort into this one-time task.